PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/springcloud/deploy/SpringCloudDeployConfiguration.java

```diff
@@ -107,7 +113,7 @@ public String getAppName() {
     }
 
     public RuntimeVersion getRuntimeVersion() {
-        return springCloudModel.getRuntimeVersion();
+        return RuntimeVersion.fromString(springCloudModel.getRuntimeVersion());
     }
```

Comment:
null check for springCloudModel.getRuntimeVersion ?

Reply:
Added empty string check